frontend/src/components/ServiceProvider.js

```diff
@@ -102,49 +159,61 @@ class ServiceProvider extends React.Component {
                     </Toolbar>
                 </AppBar>
                 <div className="serviceProvider content">
-                    <div className="contentTitle">
-                        <span>{this.state.name}</span>
-                        <BookmarkIcon
-                            isBookmarked={this.state.isBookmarked}
-                            handleBookmarkClick={this.handleBookmarkClick}
-                        />
-                    </div> 
+                    <div className="contentHeader">
+                        <div className="contentTitleContainer">
```

Comment:
Material UI has `<Table>` that may be better suited for this. I haven't looked too deeply into it so it's your call.